build.sbt

```diff
@@ -286,6 +245,87 @@ lazy val rspaceBench = (project in file("rspace-bench"))
   .enablePlugins(JmhPlugin)
   .dependsOn(rspace)
 
+lazy val deployment = (project in file("deployment"))
+  .enablePlugins(sbtdocker.DockerPlugin, RpmPlugin, DebianPlugin, JavaAppPackaging)
+  .settings(commonSettings: _*)
+  .settings(
+    version := (node / version).value,
+    name := "rnode",
+    deployJar := assembly.value,
+    deployDeb := (Debian / packageBin).value,
+    deployRpm := (Rpm / packageBin).value,
```

Comment:
TBH, I'm not so keen having our own special aliases - I actually prefer the fact that we use standard `sbt-native-packager` commands and wish we did the same for our docker builds as well.  The fewer "special" commands I need to know for our project, the better...

Reply:
Ok, I'll remove them